usr/src/test/zfs-tests/tests/functional/cli_root/zpool_labelclear/labelclear.cfg

```diff
@@ -0,0 +1,26 @@
+#! /usr/bin/ksh -p
+#
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Nexenta Systems, Inc.
+#
+
+. $STF_SUITE/include/libtest.shlib
+
+typeset LABELCLEAR="$ZPOOL labelclear"
+typeset LABELREAD="$ZDB -lq"
```

Comment:
The labelclear test failures are due to the use of $ZPOOL, $ZDB etc. (here and throughout). The test suite now uses the simple commands (zpool, zdb) instead. See illumos issue 7290.

Apart from that, tests LGTM